.eslintrc

```diff
@@ -1,20 +1,22 @@
 {
-  "ecmaFeatures": {
-    "jsx": true,
-    "modules": true
-  },
+  "extends": "eslint-config-airbnb",
   "env": {
     "browser": true,
     "mocha": true,
     "node": true
   },
-  "parser": "babel-eslint",
```

Comment:
As @emmenko [said](https://github.com/gaearon/redux/pull/121#discussion_r32721792) we did not use `block-scoped-var` rule before.